replica/backup.go

```diff
@@ -222,18 +241,27 @@ func sendToCreateHole(f types.DiffDisk, offset int64, len int64) {
 	HoleCreatorChan <- hole
 }
 
-//Preload creates a mapping of block number to fileIndx (d.location).
-//This is done with the help of extent list fetched from filesystem.
-//Extents list in each file is traversed and the location table is updated
+// shouldCreateHoles returns bool which is used to as prerequisit for
+// punching holes.
+func shouldCreateHoles() bool {
```

Comment:
Lets add a TODO here to revisit this function in case Open Call is optimized in future, where the replica keeps itself open and waits for connection from controller.